packages/examples/packages/dialogs/src/components/require-scroll-content.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Box,
+  Button,
+  Container,
+  Footer,
+  Text,
+  type SnapComponent,
+} from '@metamask/snaps-sdk/jsx';
+
+/**
+ * Long content that requires scrolling.
+ *
+ * @returns The long content component.
+ */
+export const RequireScrollContent: SnapComponent = () => (
+  <Container>
+    <Box direction="vertical">
```

Comment:
`vertical` is the default.
```suggestion
    <Box>
```